bigbluebutton-tests/playwright/user/lockViewers.js

```diff
@@ -141,25 +146,26 @@ class LockViewers extends MultiUsers {
     await this.modPage.waitAndClick(e.applyLockSettings);
     // join second user
     await this.initUserPage2();
-    await this.modPage.getLocator(e.userListItem).filter({ hasNotText: this.userPage2.username }).click();
-    await this.modPage.waitAndClick(e.unlockUserButton);
-    await this.userPage.getLocator(e.userListItem).filter({ hasText: this.userPage2.username }).click();
-    await this.userPage.waitAndClick(`${e.startPrivateChat} >> visible=true`);
+    await this.modPage.waitAndClick(`${e.moreOptionsUserItemButton}>>nth=0`);
```

Comment:
in cases like this, where there are 2 items with the same selector, avoid using `>>nth=` and prefer filtering like`getLocator().filter().locator().click` as it's more  precise and readable